winui/Kanban/cards.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: post
+title: Cards in WinUI Kanban control | Syncfusion
+description: This section describes about cards in Syncfusion WinUI Kanban (SfKanban) control, its elements and more.
+platform: winui
+control: SfKanban
+documentation: ug
```

Comment:
Add kanban

https://help.syncfusion.com/winui/overview#controls-list

Reply:
Need to add GettingStarted link, once published in live, we will add this.